src/go/plugin/go.d/config/go.d/snmp.profiles/default/3com-huawei.yaml

```diff
@@ -14,6 +14,7 @@ metrics:
         OID: 1.3.6.1.4.1.43.45.1.6.1.1.1.3
         description: Cpu usage percentage
         unit: "%"
+        family: "Network/Interfaces/CPU"
```

Comment:
Looks wrong. Are you sure this is the CPU of the network interfaces? The same goes for `Network/Interfaces/{Memory,Aruba Fan, Etc}`.

Reply:
This is the output of the model, as I said, I did not edit something

Reply:
MIB browser says 
![image](https://github.com/user-attachments/assets/c29e8365-01d4-4a91-9e21-8b1b26b8b006)

Reply:
It is wrong. Can you redo it?

Reply:
define redo, the model will never be 100% correct

Reply:
ideally we would review each YAML by hand, ensuring they are in good shape

Reply:
If we don't want to have such errors, then we can have the prior way of handling families, and roll out one-by-one updates to YAML files. Let's see what everyone thinks

Reply:
> define redo

Fix obvious mistakes that you can a person can spot right away.

Reply:
>Fix obvious mistakes that a person can spot right away.

Come on, it is not right away, you need to research the OID most of the times

>Is my understanding of family: Network/Interfaces/CPU` correct? Does this family mean that this metric is the CPU utilization of a network interface?

I was not sure, and it actually it is not, also the description seems off.

also we are talking about 150 yaml files, it is not realistic to review all of them in a short amount of time.

we will of course review them, it is a manner of if we merge this before review, or if we halt it until everything is reviewed (super non-trivial amount of time, 150 files times at least 10 metrics per file that's a lot of manual work)

Reply:
> also we are talking about 150 yaml files, it is not realistic to review all of them in a short amount of time

I can do a quick review - just glance at all the changes and see if anything seems wrong. I just did and asked to fix an issue.

Reply:
> also we are talking about 150 yaml files

Can we review fewer than 150 files if you update 150 files? Can you share your plan? How do we review them if you don't want to be reviewed now?

Reply:
As I see it (if it is only me reviewing and you approving my changes)

Once we say that the collector can be used by testers (users that want it asap) we go into the review phase. I don't know how much time it will take, but using the MIB browser we should evaluate the metrics (maybe only single metrics, don't care about tables for now).

Alphabetic seems best here. I don't feel it will be called a day, it will be the same effort as the integrations, we made something and then went in and fixed almost all. wdyt?

Reply:
if users have specific devices they use we can start by reviewing the yamls that concern these sysobjectIDs first.

Reply:
I am saying this, because there are more stuff to fix or add, we might want some mapping on contexts with metrics (ifnumber from everything has the same context) etc which are all manual edits.

Reply:
> Once we say that the collector can be used by testers (users that want it asap) we go into the review phase.

I remember I reviewed the unit's PR and noted some obvious errors there. The review wasn't addressed. Why do we need "testers" to fix errors we are aware about?